data/actionsmap/yunohost.yml

```diff
@@ -226,6 +226,80 @@ user:
                 username:
                     help: Username or email to get information
 
+#############################
+#           Alias            #
+#############################
+alias:
+    category_help: Manage alias
+    actions:
+
+        ### alias_list()
+        list:
+            action_help: List alias
+            api: GET /alias
+            configuration:
+                authenticate: all
+                authenticator: ldap-anonymous
+            arguments:
+               --fields:
+                    help: fields to fetch
+                    nargs: "+"
+               -f:
+                    full: --filter
+                    help: LDAP filter used to search
+               -l:
+                    full: --limit
+                    help: Maximum number of alias fetched
+                    type: int
+               -o:
+                    full: --offset
+                    help: Starting number for alias fetching
+                    type: int
+
+        ### alias_create()
+        create:
+            action_help: Create alias
+            api: POST /alias
+            configuration:
+                authenticate: all
+            arguments:
+                alias:
+                    help: The unique alias name to create
+                    extra:
+                        required: True
+                        pattern: *pattern_email
+                -f:
+                    full: --mailforward
+                    help: List of email to forward, separated by commas without space
+                    extra:
+                        ask: ask_alias_mailforward
+                        required: True
+                        pattern: &pattern_listemail
+                            - !!str ^([\w.-]+(\+[\w.-]+)*@([^\W_A-Z]+([-]*[^\W_A-Z]+)*\.)+([^\W\d_]{2,}),?)+$
```

Comment:
also mirroring the user command. While I do agree, I think that's the topic of another PR/discussion.